packages/ckeditor5-dev-release-tools/tests/tasks/preparerepository.js

```diff
@@ -0,0 +1,237 @@
+		} );
+
+		it( 'should use the "packagesDir" option if provided instead of the default "packages" value', async () => {
+			stubs.fs.readdir.withArgs( 'current/working/dir/custom/packages/dir' ).resolves( packages );
+			options.packagesDir = 'custom/packages/dir';
+
+			await prepareRepository( options );
+
+			expect( stubs.fs.ensureDir.callCount ).to.equal( 1 );
+			expect( stubs.fs.ensureDir.getCall( 0 ).args.length ).to.equal( 1 );
+			expect( stubs.fs.ensureDir.getCall( 0 ).args[ 0 ] ).to.equal( 'current/working/dir/release/custom/packages/dir' );
+		} );
+
+		it( 'should write the root package "package.json" file', async () => {
+			delete options.pkgJsonContent;
+
+			await prepareRepository( options );
+
+			expect( stubs.fs.writeJson.callCount ).to.equal( 1 );
+			expect( stubs.fs.writeJson.getCall( 0 ).args.length ).to.equal( 3 );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 0 ] ).to.equal( 'current/working/dir/release/package.json' );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 1 ] ).to.deep.equal( {
+				name: null,
+				version: null,
+				description: null,
+				keywords: null,
+				engines: null,
+				author: null,
+				license: null,
+				homepage: null,
+				bugs: null,
+				repository: null
+			} );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 2 ] ).to.deep.equal( {
+				spaces: 2,
+				EOL: '\n'
+			} );
+		} );
+
+		it( 'should include provided values in the root package "package.json" file', async () => {
+			await prepareRepository( options );
+
+			expect( stubs.fs.writeJson.callCount ).to.equal( 1 );
+			expect( stubs.fs.writeJson.getCall( 0 ).args.length ).to.equal( 3 );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 0 ] ).to.equal( 'current/working/dir/release/package.json' );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 1 ] ).to.deep.equal( {
+				name: 'CKEditor5',
+				version: null,
+				description: 'Foo bar baz.',
+				keywords: [ 'foo', 'bar', 'baz' ],
+				engines: null,
+				author: null,
+				license: null,
+				homepage: null,
+				bugs: null,
+				repository: null
+			} );
+			expect( stubs.fs.writeJson.getCall( 0 ).args[ 2 ] ).to.deep.equal( {
+				spaces: 2,
+				EOL: '\n'
+			} );
+		} );
+
+		it( 'should copy specified files of the root package', async () => {
+			await prepareRepository( options );
+
+			expect( stubs.fs.copy.callCount ).to.equal( 4 );
+
+			expect( stubs.fs.copy.getCall( 0 ).args.length ).to.equal( 2 );
+			expect( stubs.fs.copy.getCall( 0 ).args[ 0 ] ).to.equal( 'current/working/dir/src' );
+			expect( stubs.fs.copy.getCall( 0 ).args[ 1 ] ).to.equal( 'current/working/dir/release/src' );
+
+			expect( stubs.fs.copy.getCall( 1 ).args.length ).to.equal( 2 );
+			expect( stubs.fs.copy.getCall( 1 ).args[ 0 ] ).to.equal( 'current/working/dir/CHANGELOG.md' );
+			expect( stubs.fs.copy.getCall( 1 ).args[ 1 ] ).to.equal( 'current/working/dir/release/CHANGELOG.md' );
+		} );
+
+		it( 'should copy files of all packages by default', async () => {
```

Comment:
By default, it should not copy anything. A user must define the package's directory, and then the script can take it.